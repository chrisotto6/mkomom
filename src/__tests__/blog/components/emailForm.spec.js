import React from 'react'
import { create } from 'react-test-renderer'
import { StaticQuery } from 'gatsby'
import EmailForm from '../../../blog/components/emailForm'

beforeEach(() => {
  StaticQuery.mockImplementationOnce(({ render }) =>
    render({
      site: {
        siteMetadata: {
          mailChimpUrl: `www.mailchimp.com`,
          mailChimpToken: `token`,
        },
      },
    })
  )
})

describe('EmailForm', () => {
  it('renders correctly', () => {
    const tree = create(<EmailForm />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
