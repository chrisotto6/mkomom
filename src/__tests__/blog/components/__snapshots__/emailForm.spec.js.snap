// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailForm renders correctly 1`] = `
<div
  className="w-100 bg-light-gray pa3"
>
  <div
    className="mb4"
  >
    <span
      className="f3 tc db display dark-gray mb2"
    >
      Subscribe For More
    </span>
    <span
      className="f5 tc db serif lh-copy"
    >
      Get updates delivered weekly!
    </span>
  </div>
  <form
    action="www.mailchimp.com"
    className="w-100 db flex flex-wrap items-center justify-around"
    method="POST"
  >
    <input
      className="ba b--dark-gray bg-white pa2 sans-serif w-100 mb2"
      name="EMAIL"
      placeholder="Your Email"
      type="email"
    />
    <input
      className="dn"
      name="token"
      tabIndex="-1"
      type="text"
      value=""
    />
    <button
      className="ba b--dark-gray bg-dark-gray near-white sans-serif tracked ttu pv2 w-100 pointer"
      type="submit"
    >
      Sign Up
    </button>
  </form>
</div>
`;
